.github/workflows/ci.yml

```diff
@@ -224,54 +217,15 @@ jobs:
       docker-cache: ${{ needs.build-info.outputs.docker-cache }}
       disable-airflow-repo-cache: ${{ needs.build-info.outputs.disable-airflow-repo-cache }}
 
-  wait-for-ci-images:
```

Comment:
No more need to wait for images! All of them are build in "build" step in the same workflow so we can rely on regular "depends-on".